### DISCUSSION TITLE
[Feature] Official build script / Repository / Deb

### DISCUSSION THREAD

**Discussion by MoldyTaint** [immich-app/immich#2968] [Feature Request]
### The feature

I love docker but have seen myself moving off to more LXC based containers. Immich is the cream of the crop when it comes to self hosted imaging and man do yall do it well. 

Getting a repository for Debian installs or just a bed package would be the best thing since duct tape was invented

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
I believe there is a community-built snap package.

**Comment by DocDrydenn** [comment]
+1

I would much rather run this as a Proxmox LXC instead of docker or a snap.

**Reply by DocDrydenn** [reply]
With a detailed non-docker build, we could get this added to tteck's [Proxmox LXC scripts repo](https://github.com/tteck/Proxmox) (and on his [website](http://helper-scripts.com/).)

**Comment by DimaV83** [comment]
is this dead? Or was anyone able to install Immich as a Proxmox LXC container? (without the docker)

**Comment by NegaDuck420** [comment]
+1 - really like bare LXC installs compared to docker

**Comment by dylanReinsma** [comment]
+1

**Comment by loeeeee** [comment]
Guys, I made a guide on how to install Immich in LXC without any docker with CUDA support. You may go and check it out! [immich-in-lxc](https://github.com/loeeeee/immich-in-lxc)
